geometric/tests/test_run_json.py

```diff
@@ -0,0 +1,50 @@
+"""
+A set of tests for using the QCEngine project
+"""
+
+import numpy as np
+import json, os, shutil
+from . import addons
+import geometric
+
+@addons.using_qcengine
+@addons.using_rdkit
+def test_run_json():
+    # create a test input json file
+    qc_schema_input = {
+        "schema_name": "UFF gradient",
+        "schema_version": "v0.1",
+        "molecule": {
+            "geometry": [
+                0.0, 0.0, -0.1294769411935893, 0.0,
+               -1.494187339479985, 1.0274465079245698,
+                0.0, 1.494187339479985, 1.0274465079245698
+            ],
+            "symbols": ["O", "H", "H"],
+            "connectivity": [[0, 1, 1], [0, 2, 1]]
+        },
+        "driver": "gradient",
+        "model": {
+            "method": "UFF",
+            "basis": None
+        },
+        "keywords": {},
+        "program": "rdkit"
+    }
+
+    test_input = {
+        "schema_name": "qc_schema_optimization_input",
+        "schema_version": 1,
+        "geometric_options": {
+            "coordsys": "tric",
+            "conv": 1.e-7
+        },
+        "input_specification": qc_schema_input
+    }
+    os.mkdir('run_json.tmp')
+    os.chdir('run_json.tmp')
+    json.dump(test_input, open('in.json','w'), indent=2)
+    geometric.run_json.geometric_run_json('in.json', 'out.json')
+    os.chdir('..')
+    #shutil.rmtree('run_json.tmp')
```

Comment:
The tmp folder is not deleted here for double checking the results. We should uncomment this line later to clean up the test results.

Reply:
As a note we will need a way to pass the JSON to a function without writing to a file. It is fine to have a function that read/writes the JSON, but for our purposes completely in memory is the way to go.

We should also test the results of the geometry optimization similar to that of the current RDKit test.

Reply:
By JSON do you mean the string, or the dictionary? I can quickly make a small change to separate the file opening out from this function.

Reply:
Just the dictionary, so not technically JSON.

Reply:
The checking of final geometry in Bohr is added.